vendor/fbtransform/transforms/xjs.js

```diff
@@ -11,6 +11,51 @@
 var Syntax = require('jstransform').Syntax;
 var utils = require('jstransform/src/utils');
 
+function commaAfterLastParen(value) {
+    var state = "normal";
+    var commaPos = 0;
+    for (var i=0;i<value.length;++i) {
+        if (state === "normal") {
```

Comment:
Couple small style nits (can fixup later with anything else that might come):
- 2 space indent (match the rest of the files)
- spaces around operators (`for (var i = 0; i < value.length; ++i)`)
- use `===` and semicolons consistently. The line below this is missing the former. Line 28 the latter.